models/Shape/SquareFactory.java

```diff
@@ -1,5 +1,33 @@
 package models.Shape;
 
 final public class SquareFactory {
+    public double height;
+    public double width;
+    public double area ;
+
+    public SquareFactory() {
```

Comment:
What if we wanted to create a circle instead? Can this class be modified to support more than one shape? Does this work better as ShapeFactory?